sdk/smx.ts

```diff
@@ -79,8 +83,8 @@ export class SMXStage {
 
   public readonly inputState$: Bacon.EventStream<boolean[]>;
   public readonly deviceInfo$: Bacon.EventStream<SMXDeviceInfo>;
-  public readonly configResponse$: Bacon.EventStream<SMXConfig>;
-  public readonly testDataResponse$: Bacon.EventStream<SMXSensorTestData>;
+  public readonly configResponse$: Bacon.EventStream<SMXConfig | undefined>;
+  public readonly testDataResponse$: Bacon.EventStream<SMXSensorTestData | undefined>;
```

Comment:
It feels weird to have these emit undefined. We may want to find a better way to contain the bail out cases below.